src/cr/cube/cube.py

```diff
@@ -132,6 +132,16 @@ def population_fraction(self):
         """
         return self._cubes[0].population_fraction
 
+    @lazyproperty
+    def total_n(self):
```

Comment:
The name `total` is a little bit ambiguous here. It doesn't say explicitly what it is. Maybe consider something like num_rows perhaps? I'm not against `total`, I'd just like more explicitness. Like, is it the total number of responses considered? Maybe just add that in the docstring...

Reply:
ok